src/pages/Missions/Missions.tsx

```diff
@@ -0,0 +1,199 @@
+import FoxArmyBg from 'assets/foxarmy-bg.png'
+import FoxAtarBg from 'assets/foxatar-card-bg.png'
+import SponsorBg from 'assets/mission-sponsor-bg.jpg'
+import OptimismBg from 'assets/op-card-bg.png'
+import YatBg from 'assets/yat-mission-bg.png'
+import { Main } from 'components/Layout/Main'
+import { SEO } from 'components/Layout/Seo'
+import { ScrollCarousel } from 'components/ScrollCarousel/ScrollCarousel'
+import { RawText } from 'components/Text'
+import { FOX_MISSION_REQUEST_PAGE } from 'pages/Missions/constants'
+
+import type { MissionProps } from './Mission'
+import { Mission } from './Mission'
+
+export const Missions = () => {
+  const translate = useTranslate()
+  const missionItems: MissionProps[] = useMemo(() => {
+    return [
+      {
+        title: translate('missions.foxatar.title'),
+        subtitle: translate('missions.foxatar.subtitle'),
+        buttonText: translate('missions.foxatar.cta'),
+        coverImage: FoxAtarBg,
+        onClick: () => window.open('https://app.mercle.xyz/shapeshift/events'),
+      },
+      {
+        title: translate('missions.optimism.title'),
+        subtitle: translate('missions.optimism.subtitle'),
+        buttonText: translate('missions.optimism.cta'),
+        coverImage: OptimismBg,
+        onClick: () => window.open('https://rewards.shapeshift.com/optimistic-fox-1'),
+        endDate: '2023-06-30 08:00 AM',
+      },
+      {
+        title: translate('missions.foxArmy.title'),
+        subtitle: translate('missions.foxArmy.subtitle'),
+        buttonText: translate('missions.foxArmy.cta'),
+        coverImage: FoxArmyBg,
+        onClick: () => window.open('https://x.postmint.xyz/'),
+        startDate: '2023-06-06 17:00 UTC',
+        endDate: '2023-06-13 17:00 UTC',
+      },
+      {
+        title: translate('missions.yat.title'),
+        subtitle: translate('missions.yat.subtitle'),
+        buttonText: translate('missions.yat.cta'),
+        coverImage: YatBg,
+        onClick: () => window.open('https://fantasy.y.at/invite/yduad7mm'),
+      },
+    ]
+  }, [translate])
+
+  const renderMissions = useMemo(() => {
+    const now = dayjs()
+    const groupedMissions: {
+      future: MissionProps[]
+      past: MissionProps[]
+      active: MissionProps[]
+    } = missionItems.reduce(
+      (groups, mission) => {
+        const start = dayjs(mission.startDate)
+        const end = dayjs(mission.endDate)
+        if (now.isBefore(start) || !mission.onClick) {
+          groups.future.push(mission)
+        } else if (now.isAfter(end)) {
+          groups.past.push(mission)
+        } else {
+          groups.active.push(mission)
+        }
+        return groups
+      },
+      { future: [] as MissionProps[], past: [] as MissionProps[], active: [] as MissionProps[] },
+    )
+
+    return (
+      <Container maxWidth='full' display='flex' flexDir='column' gap={12} px={{ base: 0, md: 6 }}>
+        {groupedMissions.active.length > 0 && (
```

Comment:
Not worth blocking, but in the future this would read a lot easier if we used a element constant for each of these conceptual elements (e.g. `ActiveMissions`, `MissionsComingSoon`, etc).

Reply:
Resolving so we can merge this - cc @reallybeard